### ISSUE TITLE
Current version isn't downloading

### ISSUE THREAD

**Issue by jleighherzog** [CDL-Dryad/dryad-product-roadmap#1765]
When the "Download dataset" button is selected, the previous version is downloaded vs the most recent version. 
Please investigate these data submissions:  

Title: Origin of the species richness and composition of Caribbean aquatic entomofauna
DOI: 10.5061/dryad.vhhmgqntx

Title: Forecasting suppression of invasive Sea Lamprey in Lake Superior: data and code for Bayesian forecast model
DOI: 10.5061/dryad.69p8cz946

**Comment by sfisher** [comment]
There appear to be extra versions in Merritt from around the time they did some bad updates to their system.  They appear to have added versions even though they indicated to us that submissions failed and did not go in.

This query seems to give info to compare with Merritt.  Most of the extra version seem to have been created around May 3rd with those issues they were having.

```
SELECT res.id, res.updated_at, res.title, res.tenant_id, res.meta_view, res.file_view,
	ver.version, ver.merritt_version, res.download_uri, res_state.resource_state, sub_log.`created_at` as sub_date, sub_log.`archive_response`, sub_log.`archive_submission_request`
FROM stash_engine_identifiers ids
JOIN stash_engine_resources res
ON ids.id = res.identifier_id
JOIN stash_engine_versions ver
ON res.id = ver.`resource_id`
JOIN stash_engine_resource_states res_state
ON res.id = res_state.resource_id
JOIN `stash_engine_submission_logs` sub_log
ON res.id = sub_log.`resource_id`
WHERE ids.identifier = '10.5061/dryad.vhhmgqntx';
```

**Comment by sfisher** [comment]
There were 57 that had problems submitting around May 3rd because of some Merritt issues. Some of these had extra versions added in Merritt even though they showed errored and that they hadn't submitted.  I checked with them and they said there weren't problems aside from the extra version.  I updated the version numbers to skip the extra version.  Not all 57 of these had this problem but probably about 30% did.

Let me know if you see any more that are like this, but I believe they will all be fixed now and were all related to something odd about the Merritt repository that was a one-time problem that they've since fixed.
